src/components/button-toggle/button-toggle-classic.style.js

```diff
@@ -41,15 +50,30 @@ const StyledButtonToggleClassicLabel = ({ theme }) => theme.name === THEMES.clas
   `};
 `;
 
-const StyledButtonToggleClassicIcon = ({ theme }) => theme.name === THEMES.classic && css`
+const iconFontSizes = {
+  smallIcon: 16,
+  largeIcon: 60
+};
+
+const StyledButtonToggleClassicIcon = ({ theme }) => isClassic(theme) && css`
   display: inline;
 
   ${({ buttonIconSize }) => buttonIconSize === 'large' && css`
     display: block;
 
-    .carbon-icon {
+    ${StyledIcon} {
       margin-bottom: 0;
+      ::before {
+        font-size: ${`${iconFontSizes[`${buttonIconSize}Icon`]}px`};
+        line-height: ${`${iconFontSizes[`${buttonIconSize}Icon`]}px`};
```

Comment:
could you extract that font size selection to a method to make it more readable?

Reply:
Yeah will do